<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>341</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>日期时间</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_SetDate">
       <property name="text">
        <string>设置日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_SetTime">
       <property name="text">
        <string>设置时间</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_Time">
       <property name="inputMask">
        <string>99:99:99;0</string>
       </property>
       <property name="text">
        <string>::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日期时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_GetNowTime">
       <property name="text">
        <string>获取当前时间</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_Date">
       <property name="inputMask">
        <string>9999/99/99;0</string>
       </property>
       <property name="text">
        <string>//</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_DateTime">
       <property name="inputMask">
        <string>9999/99/99 99:99:99;0</string>
       </property>
       <property name="text">
        <string>// ::</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_DisplayStr">
       <property name="text">
        <string>字符串显示</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>日期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_SetDateTime">
       <property name="text">
        <string>设置时间与日期</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>541</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>日历</string>
   </property>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Search">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>9999年99月99日;0</string>
    </property>
    <property name="text">
     <string>年月日</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查找日期：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>290</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>979</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>981</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
